genrl/trainers/offpolicy.py

```diff
@@ -161,12 +160,15 @@ def train(self) -> None:
 
             state = next_state.detach().clone()
 
-            if self.check_game_over_status(timestep, done):
+            if self.check_game_over_status(done):
                 self.noise_reset()
 
                 if self.episodes % self.log_interval == 0:
                     self.log(timestep)
 
+                if self.episodes % self.epochs == 0:
+                    break
+
```

Comment:
Does this completely resolve the issue? 

How are the total timesteps to run for now calculated?

Reply:
See now we have the option to either restrict the total number of timesteps or the total number of episodes. Calculating number of timesteps from the number of episodes is what I wanted to remove because there's no direct correlation.

Reply:
Ok. That makes sense